cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/ExecutorConfig.java

```diff
@@ -62,6 +65,9 @@ public static ExecutorConfig create(
             @JsonProperty("cache_fetched_uris") boolean cacheFetchedUris)
             throws URISyntaxException, UnsupportedEncodingException {
 
+        if (libmesosLocation == null || StringUtils.isEmpty(libmesosLocation)) {
```

Comment:
Could have a comment here explaining that this check is for compatibility with upgrades from 1.0.22